docker/dev.yml

```diff
@@ -5,21 +5,14 @@ services:
       - "${POSTGRES_PORT:-5512}:5432"
     volumes:
       - ./data:/mnt/data
-    networks:
-      - po_default
 
   query:
     build: .
     image: openfoodfacts-query:dev
     environment:
-      # Use Product Opener's MongoDB
+      # Use Product Opener's MongoDB and REDIS
       - MONGO_URI=mongodb://mongodb:27017
+      - REDIS_URL=redis://redis:6379
```

Comment:
or is it because you use default .env for dev without docker ?

Reply:
Yes. For local dev we want "redis://localhost:6379" for example but inside the container we need to refer to the other container's name rather than localhost.